viz-lib/src/lib/value-format.js

```diff
@@ -13,7 +16,15 @@ export function createTextFormatter(highlightLinks) {
   if (highlightLinks) {
     return value => {
       if (isString(value)) {
-        value = value.replace(urlPattern, '$1<a href="$2" target="_blank">$2</a>');
+        const Link = visualizationsSettings.LinkComponent;
+        value = value.replace(urlPattern, (unused, prefix, href) => {
+          const link = ReactDOMServer.renderToStaticMarkup(
```

Comment:
I wonder if this might have performance implications.

Reply:
It may be a little bit slower, but if the Link component will stay simple - it shouldn't be a problem. In addition we got a safe html here - we didn't escape `href` before. DOMPurify was doing its job, but now it will be less chance that malformed url will break markup.